src/service/test/media.rs

```diff
@@ -116,8 +119,97 @@ fn thumbnail_bad_path_returns_not_found() {
 
 	let path: PathBuf = ["not_my_collection"].iter().collect();
 
+	let size = None;
 	let pad = None;
-	let request = protocol::thumbnail(&path, pad);
+	let request = protocol::thumbnail(&path, size, pad);
 	let response = service.fetch(&request);
 	assert_eq!(response.status(), StatusCode::NOT_FOUND);
 }
+
+#[test]
+fn thumbnail_default_size() {
```

Comment:
I think we can compact these tests into a single one by using a pattern like this:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2c8a6dcae339bac44ed8e34e347f05d0

Replacing the `a` and `b` field with a `ThumbnailSize` and expected image size, and the loop body with the actual test request and validation.